src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/settings/ItemRoute.js

```diff
@@ -37,6 +37,7 @@ export const ItemRoute = connect(state => {
 
         return request(`${CONFIG.API_INSTANCE}/search/items?q=${fullPath}`)
           .then(res => {
+            if (!res.data) return;
```

Comment:
In general we should avoid this pattern. As it swallows any potential errors on responses. e.g. 400 response with a bad request message.

Reply:
I removed line 40 and now added error handling inside the API call